packages/q-dns-client/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/natesales/q
+TERMUX_PKG_DESCRIPTION="A tiny command line DNS client with support for UDP, TCP, DoT, DoH, DoQ and ODoH."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="kay9925@outlook.com"
+TERMUX_PKG_VERSION="0.10.1"
+TERMUX_PKG_SRCURL="git+https://github.com/natesales/q"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+	termux_setup_golang
+
+	local _gitCommit=$(cd $TERMUX_PKG_SRCDIR; git rev-parse "v${TERMUX_PKG_VERSION}")
+	local _builtAt=$(date +'%FT%T%Z')
+
+  git reset --hard ${_gitCommit}
+
+	local ldflags="\
+	-w -s \
+	-X 'main.version=${TERMUX_PKG_VERSION}' \
+	-X 'main.commit=${_gitCommit}' \
+	-X 'main.date=${_builtAt}' \
+	"
+
+	export CGO_ENABLED=1
+
+	go build -o "${TERMUX_PKG_NAME}" -ldflags="$ldflags"
+}
+
+termux_step_make_install() {
+	install -Dm700 ./${TERMUX_PKG_NAME} ${TERMUX_PREFIX}/bin
```

Comment:
Perhaps it could be changed to

install -Dm700 . /${TERMUX_PKG_NAME} ${TERMUX_PREFIX}/bin/q

Reply:
From other linux distributions, you can do it.